validphys2/src/validphys/photon/compute.py

```diff
@@ -112,7 +112,7 @@ def a_em_nlo(self, q, a_ref, qref, nf):
         nd = nf - nu
         beta0 = ( -4.0 / 3 * (nl + nc * (nu * self.eu2 + nd * self.ed2)) )
         beta1 = -4.0 * ( nl + nc * (nu * self.eu2**2 + nd * self.ed2**2) )
-        lmu = np.log(q / qref)
+        lmu = np.log(q**2 / qref**2)
```

Comment:
Maybe it is slightly more efficient to multiply by 2 the result, since you never computed the squares until now :)